stories/assets/scss/_typography.scss

```diff
@@ -191,10 +192,60 @@ a {
   @extend %animate-link !optional;
 }
 
+// Documentation styles
```

Comment:
Documentation styles should be taken out from here. _typography.scss is being bundled into compiled base styles and that section is completely irrelevant to any implementation outside of Storybook. Can be incorporated into documentation.scss as it is not used outside of storybook

Reply:
Documentation styles are moved out of _typography and now set to the documentation.scss file, and now only apply to "Docs"